manager/state/raft/raft.go

```diff
@@ -1779,21 +1792,19 @@ func (n *Node) processEntry(ctx context.Context, entry raftpb.Entry) error {
 	}
 
 	if !n.wait.trigger(r.ID, r) {
+		log.G(ctx).Errorf("Unable to find wait for raft id %x", r.ID)
+
 		// There was no wait on this ID, meaning we don't have a
 		// transaction in progress that would be committed to the
 		// memory store by the "trigger" call. Either a different node
 		// wrote this to raft, or we wrote it before losing the leader
-		// position and cancelling the transaction. Create a new
+		// position and cancelling the transaction. This entry still needs
+		// to be commited since other nodes have already created a new
 		// transaction to commit the data.
-
```

Comment:
Removing this since the only way we can get here is if the wait item was removed by calling cancelAll(). Please let me know if you think otherwise @aaronlehmann thx!